doc/_includes/forward.rst

```diff
@@ -642,62 +625,60 @@ description line containing the following fields:
 Each coil description line is followed by one or more integration point lines,
 consisting of seven numbers:
 
-** <*weight*>**
-
-    Gives the weight for this integration point (last column in Tables
-    :ref:`normal_coil_descriptions` and :ref:`accurate_coil_descriptions`).
-
-** <*x/m*> <*y/m*> <*z/m*>**
-
-    Indicates the location of the integration point (fourth column in Tables
-    :ref:`normal_coil_descriptions` and :ref:`accurate_coil_descriptions`).
+- :samp:`{<weight>}`: Gives the weight for this integration point (last column
+  in Tables :ref:`normal_coil_descriptions` and
+  :ref:`accurate_coil_descriptions`).
 
-** <*nx*> <*ny*> <*nz*>**
+- :samp:`{<x/m>} {<y/m>} {<z/m>}`: Indicates the location of the integration
+  point (fourth column in Tables :ref:`normal_coil_descriptions` and
+  :ref:`accurate_coil_descriptions`).
 
-    Components of a unit vector indicating the field component to be selected.
-    Note that listing a separate unit vector for each integration points allows
-    the implementation of curved coils and coils with the gradiometer loops
-    tilted with respect to each other.
+- :samp:`{<nx>} {<ny>} {<nz>}`: Components of a unit vector indicating the
+  field component to be selected. Note that listing a separate unit vector for
+  each integration points allows the implementation of curved coils and coils
+  with the gradiometer loops tilted with respect to each other.
 
 Creating the coil definition file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------------
 
-The standard coil definition file $MNE_ROOT/share/mne/coil_def.dat
+The standard coil definition file :file:`{$MNE_ROOT}/share/mne/coil_def.dat`
 is included with the MNE software package. The coil definition file
 can be recreated with the MNE-C utility ``mne_list_coil_def``
 as follows:
 
-mne_list_coil_def --out $MNE_ROOT/share/mne/coil_def.dat
+.. code-block:: console
+
+    $ mne_list_coil_def --out $MNE_ROOT/share/mne/coil_def.dat
```

Comment:
Let's nuke this. It can live in the MNE-C manual. (In general I'm trying to get rid of any reference to a `mne_*` command and leave those to the MNE-C manual.)